mne/io/nicolet/nicolet.py

```diff
@@ -0,0 +1,207 @@
+        elif ch_name in emg or idx in emg:
+            coil_type = FIFF.FIFFV_COIL_NONE
+            kind = FIFF.FIFFV_EMG_CH
+        elif ch_name in misc or idx in misc:
+            coil_type = FIFF.FIFFV_COIL_NONE
+            kind = FIFF.FIFFV_MISC_CH
+        else:
+            coil_type = FIFF.FIFFV_COIL_EEG
+            kind = FIFF.FIFFV_EEG_CH
+        chan_info = {'cal': cal, 'logno': idx + 1, 'scanno': idx + 1,
+                     'range': 1.0, 'unit_mul': 0., 'ch_name': ch_name,
+                     'unit': FIFF.FIFF_UNIT_V,
+                     'coord_frame': FIFF.FIFFV_COORD_HEAD,
+                     'coil_type': coil_type, 'kind': kind, 'loc': np.zeros(12)}
+        info['chs'].append(chan_info)
+    return info, header_info
+
+
+class RawNicolet(_BaseRaw):
+    """Raw object from Nicolet file
+
+    Parameters
+    ----------
+    input_fname : str
+        Path to the Nicolet file.
+    montage : str | None | instance of Montage
+        Path or instance of montage containing electrode positions.
+        If None, sensor locations are (0,0,0). See the documentation of
+        :func:`mne.channels.read_montage` for more information.
+    eog : list or tuple
+        Names of channels or list of indices that should be designated
+        EOG channels. Values should correspond to the electrodes in the
+        data file. Default is None.
+    ecg : list or tuple
+        Names of channels or list of indices that should be designated
+        ECG channels. Values should correspond to the electrodes in the
+        data file. Default is None.
+    emg : list or tuple
+        Names of channels or list of indices that should be designated
+        EMG channels. If None (default), the channel names beginning with
+        ``EMG`` are used.
+    misc : list or tuple
+        Names of channels or list of indices that should be designated
+        MISC channels. Values should correspond to the electrodes in the
+        data file. Default is None.
+    stim_channel : str | int | None
+        The channel name or channel index (starting at 0).
+        -1 corresponds to the last channel (default).
+        If None, there will be no stim channel added.
+    preload : bool or str (default False)
+        Preload data into memory for data manipulation and faster indexing.
+        If True, the data will be preloaded into memory (fast, requires
+        large amount of memory). If preload is a string, preload is the
+        file name of a memory-mapped file which is used to store the data
+        on the hard drive (slower, requires less memory).
+    verbose : bool, str, int, or None
+        If not None, override default verbose level (see mne.verbose).
+
+    See Also
+    --------
+    mne.io.Raw : Documentation of attribute and methods.
+    """
+    def __init__(self, input_fname, montage, eog=None, ecg=None, emg=None,
+                 misc=None, stim_channel=-1, preload=False, verbose=None):
```

Comment:
> Yes. That's probably wise. The channel types can be inferred from the names,

that's probably dangerous to do such magic.

> but I'm still wondering what some of the channels are (pho, sp1, sp2, rs, t1, t2). I couldn't find any documentation online.

forget it. This is just one dataset. We can put it to misc with
set_channels_type.